web/gui/dashboard_info.js

```diff
@@ -4148,6 +4148,58 @@ netdataDashboard.context = {
         info: '<p>Network traffic received and sent by pgbouncer.</p><p><b>Received</b> - received from clients. <b>Sent</b> - sent to servers.</p>'
     },
 
+    // ------------------------------------------------------------------------
+    // CASSANDRA
+
+    'cassandra.client_requests_rate': {
+        info: 'Client requests received per second. Consider whether your workload is read-heavy or write-heavy while choosing a compaction strategy.'
+    },
+    'cassandra.client_requests_latency': {
+        info: 'Response latency of requests received per second. Latency could be impacted by disk access, network latency or replication configuration.'
+    },
+    'cassandra.key_cache_hit_ratio': {
+        info: 'Key cache hit ratio indicates the efficiency of the key cache. If ratio is consistently < 80% consider increasing cache size.'
+    },
+    'cassandra.key_cache_hit_rate': {
+        info: 'Key cache hit rate measures the cache hits and misses per second.'
+    },
+    'cassandra.storage_live_disk_space_used': {
+        info: 'Amount of live disk space used.'
+    },
```

Comment:
"cassandra.storage_total_disk_space_used" is missing here @thiagoftsm

Reply:
This is a table metric. The system total is the sum of `total_disk_space_used` for all tables. We removed table metrics so we no longer have `total_disk_space_used`.

Reply:
I added @shyamvalsan .